### PR TITLE
Add FI locale

### PR DESCRIPTION
**Pull request by sjuvonen**
This patch adds Finnish language support for date-fns v2.0. 

Parsing date strings is left unimplemented as it was so already in the original version, it seems. (For code-compatibility match/index.js is copied from locale en-US.)

### CODE REVIEW

File: src/locale/fi/_lib/formatLong/index.js [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
-})
+var dateFormats = {
+  full: 'eeee d. MMMM y',
+  long: 'd. MMMM y',
+  medium: 'd.L.y',
+  short: 'd.M.y'
+}
+
+var timeFormats = {
+  full: "HH.mm.ss zzzz",
```
Comment (leshakoss) [inline src/locale/fi/_lib/formatLong/index.js]: Please change to single quote characters: https://travis-ci.org/date-fns/date-fns/jobs/389398279#L494

File: src/locale/fi/_lib/localize/index.js [hunk 2]
```diff
@@ -1,21 +1,74 @@
...
 }
 
-var monthValues = {
-  short: ['tammi', 'helmi', 'maalis', 'huhti', 'touko', 'kesä', 'heinä', 'elo', 'syys', 'loka', 'marras', 'joulu'],
-  long: ['tammikuu', 'helmikuu', 'maaliskuu', 'huhtikuu', 'toukokuu', 'kesäkuu', 'heinäkuu', 'elokuu', 'syyskuu', 'lokakuu', 'marraskuu', 'joulukuu']
+var formattingDayValues = {
+  narrow: dayValues.narrow,
+  short: dayValues.short,
+  abbreviated: dayValues.abbreaviated,
+  wide: dayValues.wide.map(name => name + "na")
```
Comment (leshakoss) [inline src/locale/fi/_lib/localize/index.js]: Please use single quote characters: https://travis-ci.org/date-fns/date-fns/jobs/389398279#L497

File: src/locale/fi/index.js [hunk 3]
```diff
@@ -13,18 +13,18 @@
...
+    weekStartsOn: 1 /* Monday */,
+    firstWeekContainsDate: 4
+  }
+}
 
-// export default locale
+export default locale
 
-throw new Error('fi locale is currently unavailable. Please check the progress of converting this locale to v2.0.0 in this issue on Github: TBA')
+// throw new Error('fi locale is currently unavailable. Please check the progress of converting this locale to v2.0.0 in this issue on Github: TBA')
```
Comment (leshakoss) [inline src/locale/fi/index.js]: Please remove this comment

File: src/locale/fi/index.js [hunk 4]
```diff
@@ -13,18 +13,18 @@
  * @author Pyry-Samuli Lahti [@Pyppe]{@link https://github.com/Pyppe}
  * @author Edo Rivai [@mikolajgrzyb]{@link https://github.com/mikolajgrzyb}
```
Comment (leshakoss) [inline src/locale/fi/index.js]: Please add yourself to the list of authors

**Comment by sjuvonen** [comment]
Yes, I will try, I just didn't realize my commits were going straight up here.

**Comment by kossnocorp** [comment]
@sjuvonen thank you a lot for your work! I'm going to publish a new version with the locale really soon. I'll keep you updated.

**Pull request merged**